elasticsearch-monitoring/templates/prometheusrule.yaml

```diff
@@ -70,23 +70,23 @@ spec:
     rules:
     {{- if .Values.amazonService.enabled }}
     - alert: ElasticsearchAWSLowDiskSpace
-      expr: aws_es_free_storage_space_minimum{job="{{ .Release.Name }}-prometheus-cloudwatch-exporter"} <= 10240
+      expr: sum(label_join(aws_es_free_storage_space_minimum{job="{{ .Release.Name }}-prometheus-cloudwatch-exporter"}, "cluster", ":", "client_id", "domain_name")) by (cluster) / min(clamp_max(elasticsearch_filesystem_data_size_bytes{job="{{ .Release.Name }}-elasticsearch-exporter", es_data_node="true"}/1024/1024, 102400)) by (cluster) <= 0.1
       for: 15m
       labels:
         severity: warning
         group: persistence
       annotations:
-        description: 'AWS Elasticsearch cluster {{`{{ $labels.domain_namae }}`}} has less than 10GB left'
+        description: 'AWS Elasticsearch cluster {{`{{ $labels.cluster }}`}} has less than 10GB left'
```

Comment:
I think you need to change the description. The logic of the expr looks good but it doesn't reflect the description. I think it's more the % of storage available is less than 10%. Same for the next

Reply:
Well, yes and no. Actually the description is correct, but might not be accurate if the total disk size is less than 100GB. But in that case it'll still alert when the free disk space is less than 10GB, that's why the description is correct 😛 

The more specific alert description can be found in the runbook, but I think it's too long to put in the annotation.

Of course I'm open to suggestions :)

Reply:
Keep it simple.

> AWS Elasticsearch cluster {{`{{ $labels.cluster }}`}} is running low on disk space.

Reply:
Well the % doesn't really add much value even if it's a bit more accurate. I agree with Philip, keep it simple as it has more documentation in the runbook

Reply:
Agree. Fixed